docs/deck.md

```diff
@@ -1,13 +1,872 @@
+Slidenumbers: true
+
 # [fit] Typescript workshop
 
+### Adrian Mróź
+### Adrian Rydzyński
+  
+---
+
+# Day one
+
+* Introduction
+* Classes
+* Interfaces
+* Types Algebra
+* Generics
+
+---
+
+## [fit] Introduction
+
+---
+
+## A few words
+- interfaces
+- inline documentation
+- easier to refactor
+- backend friendly
+- less chance to make a mistake
+- and a lot more...
+
+^ nie wszystkie biblioteki są kompatybilne
+^ większa ilość kodu/plików
+
+---
+
+## Types part 1
+Type | Example
+---|---
+`boolean` | `const isDone: boolean = false;`
+`number` | `const decimal: number = 6;`
+`string` | `const color: string = "blue";`
+`array` | `const list: number[] = [1, 2, 3];`
+`tuple` | `const x: [string, number] = ["hello", 15];`
+`enum` | `enum Color {Red, Green, Blue} <br>const c: Color = Color.Green;`
+
+---
+
+## Types part 2
+Type | Example
+---|---
+`any` | `const variable: any = 4;`
+`void` | `const print = (): void => { console.log('printed') }`
+`null` | `const n: null = null; `
+`undefined` | `const u: undefined = undefined;`
+`object` | `const o: object = { prop: 0, prop1: "red" }`
+
+---
+
+## Union
```

Comment:
W sumie u mnie tez są unie. Może tutaj pominąć?

Reply:
Widzę, ze opierasz się na składni. Ja po prostu u siebie rozwinę i tyle.

Reply:
Tak, skoro o typach mowa to chciałem dodać, że jest taka opcja. Ale się nie upieram